atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java

```diff
@@ -2106,28 +2105,6 @@ public boolean performanceIsSensitiveToTombstones() {
         return true;
     }
 
-    /**
-     * Does not require all Cassandra nodes to be up and available, works as long as quorum is achieved.
```

Comment:
I think this comment is misleading. We still require all nodes to be up when dropping a table — we wait for schema agreement from all the Cassandra nodes. Let's add another comment, saying if there's one table, quorum is fine, but if there're 2 or more tables, we require all tables to be up.

Reply:
This comment was removed, so there's nothing to do here.